common-ui/src/main/res/values/widgets.xml

```diff
@@ -158,14 +158,56 @@
     </style>
 
     <!-- Buttons -->
+    <style name="Widget.DuckDuckGo.DaxButton.Contained" parent="Widget.MaterialComponents.Button">
+        <item name="android:textAppearance">?attr/textAppearanceButton</item>
+        <item name="shapeAppearance">?shapeAppearanceSmallComponent</item>
+    </style>
+
+    <style name="Widget.DuckDuckGo.DaxButton.Contained.Primary">
+        <item name="android:textColor">@color/button_primary_text_color_selector</item>
+        <item name="backgroundTint">@color/button_primary_container_selector</item>
+        <item name="rippleColor">@color/button_primary_ripple_selector</item>
+        <item name="iconTint">@color/button_primary_text_color_selector</item>
+    </style>
+
+    <style name="Widget.DuckDuckGo.DaxButton.Contained.Destructive">
+        <item name="android:textColor">@color/button_primary_text_color_selector</item>
+        <item name="backgroundTint">@color/button_destructive_container_selector</item>
+        <item name="rippleColor">@color/button_destructive_ripple_selector</item>
+        <item name="iconTint">@color/button_primary_text_color_selector</item>
+    </style>
+
+    <style name="Widget.DuckDuckGo.DaxButton.Secondary" parent="Widget.MaterialComponents.Button.OutlinedButton">
+        <item name="android:textAppearance">?attr/textAppearanceButton</item>
```

Comment:
We can remove this line. TextAppearance will be taken from the default value for Buttons like the other ones

Reply:
good call!